src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECKSTATUS = 'CHECKSTATUS';
```

Comment:
- [ ] Please, kindly follow the Ducks pattern for the value of each action like this:

```JAVASCRIPT
const CHECK_STATUS = 'bookStore/books/CHECK_STATUS';
```

That way, we will meet this requirement 💪